master/pkg/model/experiment.go

```diff
@@ -568,7 +567,7 @@ func NewTrial(
 		warmStartCheckpointID = &warmStartCheckpoint.ID
 	}
 	return &Trial{
-		RequestID:             &requestID,
+		RequestID:             nil,
```

Comment:
can you delete this field yet?  Or do you need to leave it in the DB for restorability reasons, but for new trials it is always nil?

Reply:
yeah, restore logic queries the old trial's request id.